CesiumRasterOverlays/include/CesiumRasterOverlays/WebMapTileServiceRasterOverlay.h

```diff
@@ -0,0 +1,168 @@
+#pragma once
+
+#include "Library.h"
+#include "RasterOverlay.h"
+
+#include <CesiumAsync/IAssetRequest.h>
+#include <CesiumGeometry/QuadtreeTilingScheme.h>
+#include <CesiumGeospatial/Ellipsoid.h>
+#include <CesiumGeospatial/GlobeRectangle.h>
+#include <CesiumGeospatial/Projection.h>
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+namespace CesiumRasterOverlays {
+
+class CreditSystem;
+
+/**
+ * @brief Options for tile map service accesses.
+ */
+struct WebMapTileServiceRasterOverlayOptions {
+
+  /**
+   * @brief The MIME type for images to retrieve from the server.
+   *
+   * Default value is "image/jpeg"
+   */
+  std::optional<std::string> format;
```

Comment:
Current treatment is to set default by value_or rather than value assignment. Not sure i get your point correctly.

Reply:
Yeah, that's what I'm wondering. Why use `std::optional` and `value_or`, rather than just making the default the initial value of the field in the `WebMapTileServiceRasterOverlayOptions` struct?

Reply:
I was thinking that if there's a need to parse capabilities xml, the `std::optional` could work with the `<Format>` attribute.

Reply:
Ah ok, so you can distinguish in the future between "unspecified, get it from the GetCapabilities" and "I specifically want to use image/jpeg." Makes sense.